exchanges/multipart-fetch/src/multipartFetchExchange.ts

```diff
@@ -9,7 +9,13 @@ import {
   makeFetchSource,
 } from '@urql/core/internal';
 
-export const multipartFetchExchange: Exchange = ({
+export interface MultipartFetchExchangeOpts {
+  customFileCheck?: (maybeFile: unknown | undefined) => boolean;
+}
+
+export const multipartFetchExchangeWithOptions = ({
+  customFileCheck,
+}: MultipartFetchExchangeOpts = {}): Exchange => ({
```

Comment:
other exchanges have also gone through this breaking change, but maybe it's worth considering making `multipartFetchExchangeWithOptions` the only available API and naming it `multipartFetchExchange` (i.e. making that the factory function)

That'd also give us an opportunity to maybe look at refactoring some of it, since this librar hasn't had a "true" breaking change almost since its creation https://github.com/urql-graphql/urql/blob/main/exchanges/multipart-fetch/CHANGELOG.md

Reply:
Well, I'm good with it either way, honestly. 😄

If you decide you'd prefer a breaking change to get the API cleaner, I'll happily make that adjustment.